server/api/core/import.js

```diff
@@ -439,9 +439,10 @@ Import.object = function (collection, key, object) {
       $setOnInsert: defaultValuesObject
     });
   }
-  if (this._count[this._name(collection)]++ >= this._limit) {
+  if (this._count[this._name(collection)] >= this._limit) {
     this.flush(collection);
   }
+  this._count[this._name(collection)] += 1;
```

Comment:
You've translated this correctly, but I don't think it was ever being done in the correct order. The logical way to do this is to increment the count and _then_ check whether we should flush. So I would move the `+=1` line above the `if`. (This shows the dangers of using ++ in this way. Harder to notice that the logic is wrong.)

Reply:
I've updated this in [this commit](https://github.com/reactioncommerce/reaction/pull/3599/commits/5685d3e6e084af7bd4b66dd7ed3ec7668cf87885)